update.sh

```diff
@@ -332,6 +332,9 @@ containers__run_update() {
 
     docker-compose up -d
     docker-compose ps
+
+    # wait for web container initialization and print its log
+    tail -f /home/ucrm/data/ucrm/log/ucrm/app/logs/webInitLog.log | sed '/^UCRM ready$/ q'
```

Comment:
we can't use absolute path for this, it needs to be taken from docker-compose.yml based on the volumes configuration